code/dqe/bucketcount.q

```diff
@@ -0,0 +1,19 @@
+\d .dqe
+
+bucketcount:{[agg;tn]
+  .lg.o[.Q.dd[`$string agg;`bucketcount];"Getting ",(string agg)," hourly count of rows in ",string tn];
+  (enlist tn)!"j"$value agg select rowcount:count i by 60 xbar time.minute from ?[tn;enlist(=;.Q.pf;last .Q.PV);1b;()]
+  }
+
+/- Given a table name as a symbol (tn), return the avg number of messages recieved each hour throughout the day
```

Comment:
Is the commenting here correct.
This would be the average number of messages received throughout the day based on hourly counts?

Same with the below

Reply:
Yes